lib/InternalRequest.php

```diff
@@ -7,28 +7,52 @@
 class InternalRequest {
     use Struct;
 
+    /** @var Client */
     public $client;
```

Comment:
its a pity that I spent so less time with php7 that I even dont know what the language actually supports. :-/